osquery/dispatcher/tests/scheduler.cpp

```diff
@@ -243,9 +243,8 @@ TEST_F(SchedulerTests, test_scheduler_drift_accumulation) {
   Config::get().update({{"data", config}});
 
   // Run the scheduler for 1 second with a second interval.
```

Comment:
Is this comment right? Shouldn't it be `Run the scheduler for 3 seconds with no interval.`?